### DISCUSSION TITLE
Advanced filtering with mathematical operations over the columns

### DISCUSSION THREAD

**Discussion by ivosabev** [prisma/prisma#14604] [Q&A] [answered]
How would you do something like this with Prisma filtering:

```sql
SELECT * 
FROM  orders
WHERE total - payment > 0
```

**Comment by austincrim** [comment, marked as answer]
Hey @ivosabev 👋,

Comparing columns of the same table is [not yet supported](https://github.com/prisma/prisma/issues/5048) in the Prisma API, but the team is currently in the research and design phase of this feature.

Feel free to watch that issue for updates, but you can use [`$queryRaw`](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#queryraw) in the meantime.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
